smoke-test/opennms-home/etc/poller-configuration.xml

```diff
@@ -61,6 +61,7 @@
       <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response" />
       <parameter key="rrd-base-name" value="http" />
       <parameter key="ds-name" value="http" />
+      <parameter key="use-system-proxy" value="false"/>
```

Comment:
Is there a need to set these given that the default is `false`?

Reply:
not really, I thought it might serve as an example - I will revert